R/pkg/R/SQLContext.R

```diff
@@ -206,7 +206,7 @@ setMethod("toDF", signature(x = "RDD"),
 #' It goes through the entire dataset once to determine the schema.
 #'
 #' @param sqlContext SQLContext to use
-#' @param path Path of file to read. A vector of multiple paths is allowed.
```

Comment:
Could you move this change to the planned new JIRA issue about parquetFile? Let's focus this PR on jsonFile